### DISCUSSION TITLE
Campaigns - selecting records from filter

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16727] [Feature improvement]
It is impossible to select all records from filter at once when adding relation to Campaigns module. moreover when we select all records from the screen and navigate to next set of data, the previous selection is erased (the same issue is in standard list view of records)
I post it as a future improvement, but in terms of functionality it has characteristic of bug in fact.

Best

**Comment by TonyM15** [comment]
This really needs to be addressed. Even in git.dev selecting from filter is not working. I have few thousands of Accounts to which frequently we sending emails, and selecting all at once saving lots of time
